showcase/app/controllers/page-components/advanced-table.ts

```diff
@@ -85,15 +86,30 @@ export default class PageComponentsAdvancedTableController extends Controller {
   // @ts-expect-error - we need to reevaluate how we get the data for the table demos when we break up the template files into sub components
   @deepTracked multiSelectUserData__demo4 = [...this.model.userDataDemo4];
   @tracked focusableElementsVisible = false;
-  @deepTracked thResizeHandleTable = new HdsAdvancedTableModel({
-    // @ts-expect-error - this will be fixed in HDS-5090
-    model: this.model.userDataShort,
```

Comment:
Upon further testing, this way of creating an AdvancedTableModel outside of the component wasn't actually working as expected. When I actually logged what the first column, it was undefined.

 I refactored to match how it is used in the actual component (ie. in a constructor) and didn't use the userDataShort anymore because it is not defined yet in the constructor. Now it works as expected.